modules/programs/qutebrowser.nix

```diff
@@ -131,7 +131,7 @@ in {
     };
 
     keyBindings = mkOption {
-      type = types.attrsOf (types.attrsOf types.str);
+      type = with types; attrsOf (attrsOf (separatedString " ;; "));
```

Comment:
If we make it even more complex with `types.coercedTo`, it becomes possible to use raw lists (i.e. without `mkMerge`) 😃 

```nix
attrsOf (attrsOf (coercedTo (listOf str) (concatStringsSep " ;; ") (separatedString " ;; ")))
```

`separatedString` remains needed to merge definitions like
```nix
# a.nix
keyBindings."a" = "foo";

# b.nix
keyBindings."a" = [ "bar" "baz" ];
```
The final order of commands depends on the evaluation order 🤷

Reply:
That's where we're coming from 😅 https://github.com/nix-community/home-manager/pull/3322#discussion_r1072772539

Reply:
Oops, as you can see I did not check your previous comments